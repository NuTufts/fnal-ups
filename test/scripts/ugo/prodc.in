Product:          ProdC
Flavor:           IRIX
                  AIX
Qualifiers:       
DB:               /home/t2/fagan/ups_database
Chains:           current
