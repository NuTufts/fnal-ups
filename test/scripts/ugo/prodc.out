Product:          ProdC
Flavor:           IRIX
                  AIX
Qualifiers:       
DB:               /thunder
                  	
                  /lightening
Chains:           current
